src/Messages.js

```diff
@@ -701,8 +701,8 @@ export default defineMessages({
   },
   groupDefinition: {
     id: 'groupDefinition',
-    description: 'Group definition label',
```

Comment:
We could perhaps update also message identifiers to separate them from the regular RBAC groups